### ISSUE TITLE
Entity addition MaskedList duplicate entry crash

### ISSUE THREAD

**Issue by Silfryi** [Asek3/Radium#19]
Got the same error as https://github.com/CaffeineMC/lithium-fabric/issues/356 while using PokeCube and some other mods, same type of entity MaskedList crash - this time it's a PokeCube trainer causing the crash.
Considering the above was with a zombie, I doubt it's specific to PokeCube here and just that it got caught up in other stuff
[crash-2022-03-19_00.01.03-server.txt](https://github.com/Asek3/Radium/files/8309923/crash-2022-03-19_00.01.03-server.txt)

**Comment by Asek3** [comment]
Plz check on [0.7.10](https://www.curseforge.com/minecraft/mc-mods/radium-reforged/files/3707226)

**Comment by Silfryi** [comment]
Will check on 1.18.2 in a moment without PokeCube, as it's not updated to 1.18.2 yet. Hopefully should provide the same opportunities for the crash.

**Comment by Silfryi** [comment]
Spent a bunch of time spawning in mobs and doing other stuff in 1.18.2, I think that it's been fixed to at least some degree - considering I never reported it.

**Comment by Asek3** [comment]
Ok, so for now

**Comment by drakray** [comment]
[crash-2022-08-15_00.31.53-client.txt](https://github.com/Asek3/Radium/files/9335142/crash-2022-08-15_00.31.53-client.txt)
I got this after trying to look at the Job tab on a Minecolonies citizen.

I opened a solo game, closed it, then went to my server, then crashed.
I tried going directly to my server the second time(without going in a solo world before), and did not crashed when looking a Job tab.
Without closing Minecraft, I went to the solo world, then went back on my server to look at the Job tab, and did not crash this time.
I cannot try going to server then solo and check a citizen there
